packages/p2p/stores/sendbird-store.js

```diff
@@ -13,6 +13,7 @@ export default class SendbirdStore extends BaseStore {
     @observable has_chat_error = null;
     @observable is_chat_loading = true;
     @observable should_show_chat_modal = false;
+    @observable orders_chat =  false
```

Comment:
What do we want to achieve with this observable? Can we rename it to make it clearer? 
Also missing a semi-colon. 🤔